app/src/main/java/com/onelist/service/ItemService.kt

```diff
@@ -9,13 +9,11 @@ import kotlinx.coroutines.withContext
 import retrofit2.awaitResponse
 
 interface IItemService {
-    //TODO: Create the Interface for the Item Service
-
-    //TODO: Create the method to get all the items
+    suspend fun fetchItems(): List<Item>?
 }
 
 class ItemService : IItemService {
```

Comment:
The ItemService class is implementing the IITemService interface. Classes that implement interfaces should use all the methods from the interface in order. The IItemService interface was empty so I added the method signature used in ItemService. 

Link to source: https://kotlinlang.org/docs/interfaces.html